### PR TITLE
Reworked incident CA-LosAngeles-19

### PR DESCRIPTION
**Pull request by ContributingThrowaway**
Fixed incorrect date. The incident seems to have been created in response to three separate incidents: inhumane police detentions by LAPD on June 1 (CA-LosAngeles-19), inhumane police detentions by LAPD on June 2 (added as a new incident, CA-LosAngeles-32), and inhumane police detentions by Santa Monica police on May 31 (added as a new incident, CA-SantaMonica-2). There are many alleged abuses perpetrated by police that deserved to be mentioned, but descriptions must be kept brief.

### CODE REVIEW

File: reports/California.md [hunk 1]
```diff
@@ -380,6 +380,44 @@ id: ca-losangeles-26
...
+* [Allegations by protester Laura Montilla](https://www.instagram.com/p/CBBNXXkJs0a/?igshid=jgeposybda4a)
+* [Relevant video](https://www.instagram.com/p/CBEr2emJRyJ/?igshid=oloj7lgtv81t)
+* [Local news coverage mentioning the incident](https://abc7.com/protest-arrest-george-floyd-curfews/6234180/)
+* [LA Times article mentioning both lawsuits](https://www.latimes.com/california/story/2020-06-22/complaints-lapd-abuses-mount-protest-lawsuit)
+* [UCLA disapproves of use of Jackie Robinson stadium for a field jail](https://twitter.com/UCLA/status/1268039753015812096)
+* [First response by UCLA professors, containing further allegations of misconduct by LAPD and LASD](https://twitter.com/ananyaUCLA/status/1267985147225231360)
+* [Second response by UCLA professors](https://drive.google.com/file/d/157MdwMMCEebcUyesO9HMOYK9p21q3gpE/view)
+* https://www.reddit.com/r/LosAngeles/comments/gxi8b2/how_is_noone_talking_about_this_women_from/
+* [Allegation of LAPD cutting someone's hand while trying to remove a zip tie](https://www.reddit.com/r/LosAngeles/comments/gy3e3r/the_lapd_has_been_slicing_into_protesters_hands/ft9so6j/?context=8&depth=9)
```
Comment (emarcey) [inline reports/California.md:401]: Are you asserting this comment on reddit as evidence? 

I don't think this is sufficient to qualify as evidence.

File: reports/California.md [hunk 2]
```diff
@@ -380,6 +380,44 @@ id: ca-losangeles-26
...
...ast one arrestee alleges that protesters were given explicit permission to remain past the curfew, then arrested for being out after curfew. Three arrestees describe in similar terms arrestees (at least one of whom was gender nonconforming) being sexually harrassed by officers. Many arrestees say they were released in the middle of the night, without transportation home, in unfamiliar parts of the city. Two arrestees describe in similar terms police mocking a group of detainees and refusing to release them until they were "good girls" or "reasonable." One detainee says this was due to some detainees removing zip ties using lighters because the ties were dangerously tight. Another detainee says that it was due to detainees requesting medical attention and other improvements in their conditions, which would constitute first-amendment-protected speech. One arrestee allegedly should have been exempt from the curfew as they were returning home from work. A lawsuit has been filed in connection with these and other abuses by the National Lawyers' Guild. (See also incidents ca-losangeles-19 (near-identical behavior the previous day) and ca-santamonica-2 (similar behavior in Santa Monica).)
```
Comment (emarcey) [inline reports/California.md:408]: ```Three arrestees describe in similar terms arrestees (at least one of whom was gender nonconforming) being sexually harrassed by officers.```

This sentence is a mouthful. Could you try re-working it for a little more clarity?

File: reports/California.md (hunk 1 above)
Comment (ContributingThrowaway) [reply reports/California.md:401]: About to edit. Not sure I agree that this isn't evidence -- we occasionally accept descriptions from twitter and instagram as evidence, even when they're accompanied with no picture/video, or one that doesn't directly bear on the truth of their claims. The reddit account making the allegation, u/blackwidowla, doesn't (as far as I can tell) have a history of posting content that looks like trolling or astroturfing. The account is 2 years old, and sometimes mentions being from LA. The (admittedly not very many) posts I looked over didn't mention living anywhere else. If u/blackwidowla is a troll, I admire its dedication to its craft. (Of course, that says nothing about the possibility that u/blackwidowla could be a real Angeleno who is lying. But that's a risk we run with any eyewitness description that doesn't come from someone known to be trustworthy.)

Comment (emarcey) [reply reports/California.md:401]: That's true, but the insta and Twitter posts are always tied to a real world identity. The other posts here are people with their names out there. And they have also been referenced in news articles, which I assume have a higher standard of due diligence

Comment (emarcey) [reply reports/California.md:401]: And I don't think it really adds enough to be worth the tricky situation imo

Comment (ContributingThrowaway) [reply reports/California.md]: Good points. Removed.

**Comment by ContributingThrowaway** [comment]
It looks like there may have been one bus where officers delayed processing in retaliation for removing unsafe zip ties, and another bus where officers delayed processing in retaliation for first-amendment-protected speech. Revising to reflect this.

**Comment by emarcey** [comment]
@ubershmekel something seems to be up with the validate data action. I'm not sure what's going on

**Comment by ubershmekel** [comment]
@emarcey it seems the validate data build is failing when it tries to fetch a branch from a remote repo. It was working when the branch was on the repo itself.

```
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/patch-17*:refs/remotes/origin/patch-17* +refs/tags/patch-17*:refs/tags/patch-17*
  The process '/usr/bin/git' failed with exit code 1
  Waiting 19 seconds before trying again
```

I'm not much of a git expert. But I'll try and decipher that.

**Comment by emarcey** [comment]
It seems to be working on my other prs so I think worst case scenario we just merge it and hope it's a bug bc this was opened before the action was set up

**Comment by ubershmekel** [comment]
I suspect it's an issue with remote branches that aren't on 2020PB/police-brutality the error is in `Run actions/checkout@v2`

```
Fetching the repository
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/patch-17*:refs/remotes/origin/patch-17* +refs/tags/patch-17*:refs/tags/patch-17*
  The process '/usr/bin/git' failed with exit code 1
  Waiting 19 seconds before trying again
```

Here's what it looks like when it worked at https://github.com/2020PB/police-brutality/runs/816195725

```
Fetching the repository
  /usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +refs/heads/emarcey-patch-70*:refs/remotes/origin/emarcey-patch-70* +refs/tags/emarcey-patch-70*:refs/tags/emarcey-patch-70*
```

**Comment by ubershmekel** [comment]
@emarcey 100% merge it. We'll figure this out next PR that fails.

**Comment by emarcey** [comment]
Sounds good!

**Pull request merged**